### ISSUE TITLE
hOn compatibility

### ISSUE THREAD

**Issue by lucaesse** [bstuff/haier-ac-remote#33]
Hi, new haier AC uses the App hOn instead of SmartAir2.
Is it possibile to make this plugin compatible?

**Comment by tdubinin** [comment]
поддерживаю вопрос. Как я могу помочь с доработкой поддержки приложения hOn?

**Comment by vladshup** [comment]
Тоже поддерживаю и рад бы поучаствовать.  Имею 2 шт. Haier HSU-24HFM103/R3(SDB). Немного умею пользоваться Wireshark. пишу на C, PHP и немного понимаю, что написано на javascript.

**Comment by tdubinin** [comment]
смотрел трафик через charles, очень всё замудрёно сделано, сложно воспроизвести обычными http запросами (по крайней мере для меня).

в итоге сейчас начал смотреть в сторону другого wi-fi модуля (с алика esp8266) и статьи из интернета, если получится, напишу что как.

**Comment by vladshup** [comment]
Добрый день!
Спасибо за информацию. Собственно, тоже склоняюсь попробовать на esp8266,
особенно с учетом того, что мне все-таки "роднее" majordomo. Просто обидно
- модули родные стоят. Да и писать/править прошивку сейчас напряжно -
строительство коттеджа заканчиваю (кондики в нем и стоят).
Если будут результаты, обязательно напишу.
С уважением,
Владимир.

**Comment by tdubinin** [comment]
Получилось добавить кондей в умный дом?
у меня нет, esp8266 не видит ничего от кондея(

**Comment by vladshup** [comment]
Нет, не получилось, ситуация аналогичная, а разбирать протокол сейчас нет
времени.

**Comment by tdubinin** [comment]
говорят, что модуль [Haier KZW-W002 для кондиционера](https://market.yandex.ru/product--blok-upravleniia-haier-kzw-w002-dlia-konditsionera/662131100/questions) должен подойти, и в этом случае можно будет подключить кондей уже не через hOn

**Comment by vladshup** [comment]
Добрый день!
Спасибо большое за инфо.
Тут суть была - кондиционеры (2 шт.) использовались для отопления 1-го
этажа дома, площадью 78 м. кв.
Хотелось немного сэкономить, например опуская температуру ночью до +15С, ну
и подобные штуки автоматизировать.
В итоге мы перезимовали очень комфортно, и относительно недорого даже без
дополнительного "расписания".
Сейчас очередной виток стройки, т.к. дом 2 этажа, а многое делаем своими
руками. Энтузиазм с подключением кондиционеров сильно ослаб. Возможно к
зиме вернется.
С наилучшими пожеланиями.
Владимир.
